code/api/src/migrations/20200825025333-addDescriptionColumn.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => { 
+    return queryInterface.addColumn(
```

Comment:
Migrations use the Sequelize library. Here is a link to documentation if you should need it for your own migrations: https://sequelize.readthedocs.io/en/latest/docs/migrations/